packages/flutter_secure_storage/tizen/src/flutter_secure_storage.cc

```diff
@@ -0,0 +1,83 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter_secure_storage.h"
+
+#include <ckmc/ckmc-manager.h>
+#include <ckmc/ckmc-type.h>
+
+#include <algorithm>
+
+#include "log.h"
+#include "secure_storage_util.h"
+
+FlutterSecureStorage::FlutterSecureStorage() {
+  cipher_ = std::make_unique<Cipher>("FlutterSecureStorageAesKey", 12);
+}
+
+FlutterSecureStorage::~FlutterSecureStorage() {}
+
+void FlutterSecureStorage::Write(const std::string &key,
+                                 const std::string &value) {
+  std::string encrypt = cipher_->Encrypt(value);
+
+  ckmc_raw_buffer_s write_buffer;
+  write_buffer.data = (unsigned char *)encrypt.c_str();
+  write_buffer.size = encrypt.size();
+
+  ckmc_policy_s policy = {
+      .password = nullptr,
+      .extractable = true,
+  };
+
+  ckmc_save_data(key.c_str(), write_buffer, policy);
```

Comment:
`write_buffer` needs to be freed here.

```suggestion
  ckmc_save_data(key.c_str(), write_buffer, policy);
  ckmc_buffer_free(write_buffer);
```

Reply:
`write_buffer` is not pointer. could you please check again?

Reply:
Oh, you're right!